### PR TITLE
Fixed 'blank' barcode reader bug

### PR DESCRIPTION
**Pull request by MinickLS**

### CODE REVIEW

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/InternalBarcodeScanner.js [hunk 1]
```diff
@@ -2,38 +2,36 @@
...
+* @success function(struct result) function to call on success
+* @failure function(string errorMsg) function to call on failure
+*/
 internalbarcodescanner.scanBarcode = function(success, failure) {
-   console.log('scanning');
-
    var scanner = cordova.require("cordova/plugin/BarcodeScanner");
 
    var barcodeSuccess = function(result) {
-      alert("We got a barcode\n" +
```
Comment (jeslin) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/InternalBarcodeScanner.js:11]: :+1:  :+1:  :+1:

**Comment by jeslin** [comment]
Looks good to me! Waiting on someone else to review so we can merge :)

**Comment by kaysabelle** [comment]
Looks good to me!

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/InternalBarcodeScanner.js [hunk 2]
```diff
@@ -2,38 +2,36 @@
 var resources = require('./Resources');
 var internalbarcodescanner = {};
 
+/**
```
Comment (kaysabelle) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/InternalBarcodeScanner.js:5]: Nice comments.

**Comment by kaysabelle** [comment]
Going to merge this!

**Pull request merged**